packages/story-editor/src/components/panels/design/videoOptions/videoOptions.js

```diff
@@ -75,23 +75,23 @@ function VideoOptionsPanel({ selectedElements, pushUpdate }) {
   const { isTranscodingEnabled } = useFFmpeg();
   const {
     muteExistingVideo,
-    isResourceTrimming,
+    isElementTrimming,
     isNewResourceMuting,
     canTranscodeResource,
   } = useLocalMedia(
     ({
-      state: { canTranscodeResource, isNewResourceMuting, isResourceTrimming },
+      state: { canTranscodeResource, isNewResourceMuting, isElementTrimming },
       actions: { muteExistingVideo },
     }) => ({
       canTranscodeResource,
       isNewResourceMuting,
-      isResourceTrimming,
+      isElementTrimming,
       muteExistingVideo,
     })
   );
   const resource = getCommonValue(selectedElements, 'resource');
-  const { isMuted, id: resourceId = 0 } = resource;
-  const isTrimming = isResourceTrimming(resourceId);
+  const { isMuted, id: resourceId = 0, elementId = 0 } = resource;
```

Comment:
`elementId` is not part of `resource`; it's he ID of the element itself. So this doesn't work. It will always be `0`.

`const elementId = getCommonValue(selectedElements, 'id');` would get you the ID

Reply:
Update here - https://github.com/GoogleForCreators/web-stories-wp/pull/10712/commits/7f576fd0dddf012865501affb514c3c5e7107e44